lib/ips/i2c.cpp

```diff
@@ -55,7 +55,7 @@ bool I2c::init() {
   XIic_SetStatusHandler(&xIic, this, (XIic_StatusHandler)StatusHandler);
 
   irqs[i2cInterrupt].irqController->enableInterrupt(irqs[i2cInterrupt],
-                                                    polling);
+                                                    0); //polling);
```

Comment:
Ist das noch WIP? Sollte das nicht vielleicht besser `1 ? polling : 0` lauten?

Reply:
Nein - i2c soll immer interrupts benutzen. Aktuell ist polling eine configuration der card - sollten man in der zukunft ändern. Will irgendwann eine art overlay config für die IPs ermöglichen.